### ISSUE TITLE
Outdated multisig reference

### ISSUE THREAD

**Issue by mkoura** [input-output-hk/cardano-node#1914]
**Internal**

**Summary**
After merging https://github.com/input-output-hk/cardano-node/pull/1880 the multisig reference on https://github.com/input-output-hk/cardano-node/blob/master/doc/reference/multisig.md is no longer up-to-date.

**Expected behavior**
1. description on how to construct scripts now that `cardano-cli shelley transaction build-multisig` is no longer available (how to get key hashes etc.)
2. commands are up-to-date - `transaction sign-witness` vs `transaction assemble` etc.

**Comment by dcoutts** [comment]
We reviewed how we were asking users to write their mutli-sig scripts. We had two methods:
1. using the CLI with flags to generate a mutli-sig script in JSON format
2. writing the JSON format directly

The CLI-based method was incomplete in that it did not provide the full expressive power of the script language, and making it complete while keeping it as a CLI-based scheme would make it unwieldy.

Therefore we decided to simply remove the CLI-based method and require that people write the scripts in JSON directly. This provides the full expressive power and is still pretty straightforward, and reduces the number of different ways of doing the same thing.

So yes, the multi-sig reference doc will need to be updated to remove the step that uses the CLI to creating a script JSON files and instead just says to make a script file directly.
